ionic.config.json

```diff
@@ -4,5 +4,6 @@
     "cordova": {}
   },
   "type": "ionic-angular",
-  "pro_id": "b496a218"
```

Comment:
Looks good. I still was on Ionic CLI 4.4.0, so I didn't notice the pro_id thing. Thanks!